### DISCUSSION TITLE
LogContext: Add infinite scrolling to automatically load new log lines

### DISCUSSION THREAD

**Discussion by svennergr** [grafana/grafana#56091] [Observability]
Currently the user has to click a button to load another 10 log lines into the context. This could also be done with an infinite scrolling behavior.
When the user scrolls down in the list of loglines Grafana will automatically add more loglines to the context before the user reached the end.

**Comment by svennergr** [comment]
Created https://github.com/grafana/grafana/issues/57052

**Comment by gabor** [comment]
just a note: some databases, like Elastic have the concept of pagination in them, so they are able to do something like:
1. give me all the logs between timestamps A and B, but only the first 100 items
2. then, give me all the logs between timestamps A and B, but only the items between 101 and 200
3. then, give me all the logs between timestamps A and B, but only the items between 201 and 300


( https://www.elastic.co/guide/en/elasticsearch/reference/current/paginate-search-results.html )

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
